dask_ml/wrappers.py

```diff
@@ -662,9 +663,28 @@ def _get_output_dask_ar_meta_for_estimator(model_fn, estimator, input_dask_ar):
     """
     # sklearn fails if input array has size size
     # It requires at least 1 sample to run successfully
-    ar = np.zeros(
-        shape=(1, input_dask_ar.shape[1]),
-        dtype=input_dask_ar.dtype,
-        like=input_dask_ar._meta,
-    )
+    imput_meta = input_dask_ar._meta
```

Comment:
```suggestion
    input_meta = input_dask_ar._meta
```

Is this a typo? The variable name probably makes more sense if it says "input" (instead of "imput")

Reply:
Yup. That was a typo. Thanks for the review. Pushed a fix with commit . https://github.com/dask/dask-ml/pull/889/commits/a295e39d11826c60faf43e0d1d42dc66a9a0a265